### PR TITLE
Development

### PR DESCRIPTION
**Pull request by timoj**
## Description

- Added support for MultiSafePay

## References

JIRA: [WWE-252](https://wuunder.atlassian.net/secure/RapidBoard.jspa?rapidView=19&projectKey=WWE&modal=detail&selectedIssue=WWE-252)

## Tasks

* [x] update CHANGELOG.md
* [ ] reviewed (by @Mittows   )
* [x] merge-in origin/master/development

### CODE REVIEW

File: Wuunder/Controllers/Backend/WuunderShipment.php [hunk 1]
```diff
@@ -123,7 +122,7 @@ private function getData($order_id)
             'delivery_address' => $delivery_address,
             'customer_reference' => $order->getNumber(),
             'description' => $description,
-            'value' => $value,
+            'value' => round($value),
```
Comment (Mittows) [inline Wuunder/Controllers/Backend/WuunderShipment.php]: Shouldnt this be round($value, 2)?

File: Wuunder/Views/backend/wuunder/controller/list.js [hunk 2]
```diff
@@ -5,6 +5,9 @@ Ext.define('Shopware.apps.Wuunder.controller.List', {
 
     init: function () {
         var me = this;
+        console.log("H1");
```
Comment (Mittows) [inline Wuunder/Views/backend/wuunder/controller/list.js]: Logging

File: Wuunder/Controllers/Backend/WuunderShipment.php (hunk 1 above)
Comment (timoj) [reply Wuunder/Controllers/Backend/WuunderShipment.php]: no

File: Wuunder/Views/backend/wuunder/controller/list.js (hunk 2 above)
Comment (timoj) [reply Wuunder/Views/backend/wuunder/controller/list.js]: Removed

**Pull request merged**